homeassistant/components/calendar/__init__.py

```diff
@@ -164,7 +164,7 @@ def _validate_rrule(value: Any) -> str:
     try:
         rrulestr(value)
     except ValueError as err:
-        raise vol.Invalid(f"Invalid rrule: {str(err)}") from err
+        raise vol.Invalid(f"Invalid rrule '{value}': {str(err)}") from err
```

Comment:
Exceptions have a magic method for str.
```suggestion
        raise vol.Invalid(f"Invalid rrule '{value}': {err}") from err
```